en/docs/developer-portal/test-api.md

```diff
@@ -1,30 +1,31 @@
 # Test an API
  
-Before using your API in production, you can test it using the integrated OpenAPI console in Choreo. Once the API is deployed in the required environment, an API consumer can use the application credentials, generate an OAuth 2.0 token, and invoke the API to test its functionality by providing values for the required parameters. 
+Before using your API in production, you can test it using the integrated console in Choreo. Once you deploy your API in the required environment, an API consumer can use the application credentials, generate an OAuth 2.0 token, and invoke the API to test its functionality by providing values for the required parameters. 
 
 This guide will take you through the steps to test an API in Choreo. 
 
-1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click  Developer Portal link on the Choreo console header. 
+1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click the Developer Portal link on the Choreo console header. 
 
     ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-half}
 
 2. Click **APIs**.
-3. Select an API you want to test and click **Try Out** on the card. In this case, skip step 4. Alterntively, you can click on the API and follow step 4. 
+3. Select an API you want to test and click **Try Out** on the card. In this case, skip step 4. Alternatively, you can click on the API and follow step 4. 
 4. Click **Try Out** from the left panel.
 5. Select the required environment from the **Environment** dropdown.
 
     ![Test Panel](../assets/img/developer-portal/test-api/test-panel.png){.cInlineImage-half}
 
-6. Select the relevant**Subscribed Application** from the dropdown that you subscribed the API to. 
-7. Click **Get Test Key** on the **Access Token** field to generate a test access token. 
+6. From the dropdown, select the relevant **Subscribed Application** that you subscribed the API to. To learn how to subscribe to an application, see [Manage Subscriptions](../manage-subscription/). 
```

Comment:
```suggestion
6. From the **Subscribed Application** list, select an application you want to trigger when testing the API.

     !!! info
         This list is visible only if at least one application has subscribed to your API. For more information, see [Manage Subscriptions](../manage-subscription/).
```
Suggesting this revision because this way, it is not clear what the name of the dropdown is. Also, refer [1].

Also, a user who reads this guide before reading [Manage Subscriptions](../manage-subscription/) can get confused when he/she does not see this field.

[1]  https://developers.google.com/style/ui-elements#refer-to-ui-elements